README.md

```diff
@@ -141,6 +141,22 @@ allowedTags: [],
 allowedAttributes: []
 ```
 
+### "What if I want to allow only specific values on some attributes?"
+
+When configuring the attribute in `allowedAttributes` simply use an object with attribute `name` and an allowed `values` array, this will allow any combination of the `values` in the array where the `values` are separated by a space. In the following example `sandbox="allow-forms allow-modals allow-orientation-lock allow-pointer-lock allow-popups allow-popups-to-escape-sandbox allow-scripts"` would become `sandbox="allow-popups allow-scripts"`:
+
+```js
+        allowedAttributes: {
+          iframe: [ 
+            {
+              name: 'sandbox',
+              multiple: true,
+              values: ['allow-popups', 'allow-same-origin', 'allow-scripts']
+            }
+          ]
+```
+Note: If you only want to allow a single specific attribute value, then `values` should only have a single string in the array and `multiple` should be `false`.
```

Comment:
It is not true that the `values` array must contain only one element in the absence of multiple: true.

This text makes more sense I think:

"With multiple: true, several allowed values may appear in the same attribute, separated by spaces. Otherwise the attribute must exactly match one and only one of the allowed values."

Reply:
Apologies, misunderstood the use case as we have no use case like that, reworded readme.